tests/ui/pytest.ini

```diff
@@ -0,0 +1,32 @@
+# you may not use this file except in compliance with the License.
+#
+# Copyright 2021 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+[pytest]
+norecursedirs = .git .venv venv build tests/deps node_modules package tests/expected_output_global_config_inputs_configuration_alerts tests/data/output output
```

Comment:
Do we need `tests/expected_output_global_config_inputs_configuration_alerts tests/data/output output` here?